python/ray/worker.py

```diff
@@ -685,6 +701,8 @@ def run_function_on_all_workers(self, function):
             function (Callable): The function to run on all of the workers. It
                 should not take any arguments. If it returns anything, its
                 return values will not be used.
+            run_on_other_drivers: The boolean that indicates whether we want to
+                run this funtion on other dirvers.
```

Comment:
dirvers -> drivers

Can you also explain here that this may be useful for objects shared between jobs?